.github/workflows/ci.yml

```diff
@@ -146,6 +162,10 @@ jobs:
           # The main Error Prone run
           $MAVEN ${MAVEN_TEST} -T C1 clean verify -DskipTests -P gib,errorprone-compiler \
             -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
+      - name: Clean local Maven repo
+        # Avoid creating a cache entry because this job doesn't download all dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Q: Can't we clean repo unconditionally? Since even if the cache was used this is the last command that would run on the node so clearing everything shouldn't cause any issues and will save 1 line (or 2 if you remove `name:`) per occurrence.

(Feel free to ignore - this might be an extreme comment. 😛 )

Reply:
@nineinchnick I think all Maven repo removal steps are copies of your original idea. Is this only to avoid wasting time on removal if the Maven repo was downloaded from the cache? I think in such case cache isn't updated at the end of the job.

Reply:
Oh, makes sense. We'd save time (and probably lot of it) since it's lots of small files in there.